opentelemetry/trace/main_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	trace "cloud.google.com/go/trace/apiv1"
+	"cloud.google.com/go/trace/apiv1/tracepb"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"google.golang.org/api/iterator"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func TestWriteTraces(t *testing.T) {
+	// Tests build.
+	m := testutil.BuildMain(t)
+	if !m.Built() {
+		t.Fatalf("failed to build app")
+	}
+	projectID := os.Getenv("GOLANG_SAMPLES_PROJECT_ID")
```

Comment:
Use [`testutil.SystemTest`](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/internal/testutil/testutil.go#L56) to get the project ID or skip if not available.

Reply:
done